superset/dashboards/commands/importers/v0.py

```diff
@@ -317,7 +317,10 @@ class ImportDashboardsCommand(BaseCommand):
     in Superset.
     """
 
-    def __init__(self, contents: Dict[str, str], database_id: Optional[int] = None):
+    # pylint: disable=unused-argument
+    def __init__(
+        self, contents: Dict[str, str], database_id: Optional[int] = None, **kwargs: Any
```

Comment:
is contents a flat `Dict`? also why the `kwargs` here, they receive `passwords` and `overwrite` but nothing is done with them, I'm probably missing something.

It would be nice also to add a description for the fields on the API spec:
```
                schema:
                  type: object
                  properties:
                    formData:
                      type: string
                      format: binary
                    passwords:
                      type: string
                    overwrite:
                      type: bool
```

Reply:
`contents` is a flat dict, with `filename => file_contents`.

`*kwargs` is needed because different version (`v0` vs `v1`) take different arguments, so they just discard what they don't need.

Reply:
I'll add the description to the API, I thought I had added it.

Reply:
@dpgaspar, the spec is up-to-date in `api.py`: https://github.com/apache/superset/pull/13444/files#diff-f6a877e9c3a368e3c41256960e6bddff588b7d26f5ad750d5378f2fd65722508R761-R767

Reply:
I mean a description for the `passwords`, `overwrite`, `formData` fields

Reply:
Ah, a description! Will do! :)